.github/workflows/ci.yml

```diff
@@ -9,11 +9,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Setup nightly Rust
-      run: rustup toolchain install nightly-2022-12-12 --profile minimal --target wasm32-unknown-unknown --component rust-src
     - name: Install npm deps
       working-directory: ./test
       run: npm ci
+    # For some reason isntalling the rust toolchain via wasm-pack leads to CI failures.
+    # This check will read the top level rustup-toolchain.toml and install the required stuff.
+    - name: Trigger toolchain installation
+      working-directory: ./test
+      run: cargo check
```

Comment:
Would `cargo fetch` be sufficient here? That way we would still have clear separation between "download dependencies" and "actually build" steps.

Reply:
Huh, i wasnt aware of that command. I could not find something with similar terms to install (npm) etc, and when I found out about cargo-chef I took that as a replacement for what cargo fetch seems to do.

But yeah, this seems to be sufficient according to the test commit I just did!